rate-limit-cloud-controller-api.html.md.erb

```diff
@@ -42,6 +43,22 @@ This might result in inconsistent values for the `X-RateLimit-Remaining` header
 
 When requests exceed the maximum rate limit value, CAPI returns a `429: Too Many Requests` error code.
 
+
+## <a id="v2-api"></a> Rate Limit Responses: V2 API
+
+Operators can optionally limit the number of V2 API requests. This includes all `v2/*` requests except `v2/info`. Each HTTP header includes the following information:
+
+```
+X-Ratelimit-Limit-V2-Api: 60
+X-Ratelimit-Remaining-V2-Api: 56
+X-Ratelimit-Reset-V2-Api: 1643767322
+```
+
+V2 API requests are counted identically to the general rate limit and the headers contain the same information. See table above for more details.
+When the maximum rate limit value is exceeded, CAPI returns a `429 Too Many Requests` response with the body `CF-RateLimitV2APIExceeded`.
+
+Depending on the CAPI configuration users and clients which include scope `cloud_controller.v2_api_rate_limit_exempt` can be exempted from the V2 API rate limit.
```

Comment:
How about:

"_Operators can configure CAPI to exempt users and clients with the UAA scope `cloud_controller.v2_api_rate_limit_exempt` from the V2 API rate limit._"

I guess it'd also make sense here to have a permalink to the specific bit of the capi-release spec file that can be used for this (or just directly give the name of the field here).